data/data/install.openshift.io_installconfigs.yaml

```diff
@@ -2814,8 +2914,6 @@ spec:
                       set the vnicProfileID.
                     type: string
                 required:
-                - api_vip
-                - ingress_vip
```

Comment:
The thing is, that we need a VIP. Either from the old or the new field. But declaring both fields as required, would be an issue too (because we need only one). So I declare the old and the new fields as optional, but check in the code, that at least one is set. I didn't know how to define in "yaml/json" validation that "either field x or field y has to be set".